static/src/javascripts/projects/common/views/experiments/measure-understanding.html

```diff
@@ -0,0 +1,12 @@
+<div class="understanding">
+    <div class="understanding__text">Do you feel better informed as a result of reading this piece?</div>
+    <button class="understanding__choice js-button" data-value="yes">
```

Comment:
hmm, I think the semantics of `button` is right, but you are correct the question is not read. Making it focusable should solve the problem, right? `<div tabindex=0>Do you ...`